### PR TITLE
Fixes up the README.md and adds up the contribuitors to the section

### PR DESCRIPTION
**Pull request by abhay-raj19**
enhances up the structure of Readme.md 
Added up the contributions to the section below in the readme.md

### CODE REVIEW

**Review by mveer-jaipur — COMMENTED** [review]
@hkirat Could you please review the review?

File: README.md [hunk 1]
```diff
@@ -1,31 +1,113 @@
...
+### 🐬Building up the project with Docker
 
+1. Build the Docker Image
+```markdown
+    docker-compose up
 ```
-cd daily-code
-yarn install
+
+## How to Contribute to this repository
```
[the remark this answers is not in the document]
Comment (abhay-raj19) [reply README.md:50]: Basically @mveer-jaipur it , is like restricting up user at some cases but can remove it totally ?
What's your views?

[the remark this answers is not in the document]
Comment (mveer-jaipur) [reply README.md:50]: As per the github guidelines, Open source projects should have this file to guide the developers. https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

File: README.md [hunk 2]
```diff
@@ -1,31 +1,113 @@
...
-docker compose watch
+2. Copy the env example to .env .
+```markdown
+    cd /packages/db
+    cp .env.example .env
 ```
+3. Update the .env file with the database url.
 
-    OR
+4.  Migrate up the Database and Seed the database.
```
[the remark this answers is not in the document]
Comment (vanshaj-sharma) [reply README.md:26]: it will be better if change step 5 to

**Comment by devsargam** [comment]
@abhay-raj19 can u remove the conflicts

**Comment by abhay-raj19** [comment]
@devsargam Yes Sir will fix it by today itself .

**Comment by abhay-raj19** [comment]
Hey @devsargam what i have seen is that the changes that I want to add are been added by some .
So, no issues and appreciate the changes .

**Pull request closed without merging**